packages/gluestick/src/config/__tests__/compileWebpackConfig.test.js

```diff
@@ -14,7 +19,18 @@ const loggerMock = {
   error: jest.fn(),
 };
 
+const originalProcessCwd = process.cwd.bind(process);
+
 describe('config/compileWebpackConfig', () => {
+  beforeAll(() => {
+    process.cwd = () => '.';
+  });
+
+  afterAll(() => {
+    jest.resetAllMocks();
```

Comment:
I am pretty sure we do not need this line here as we just have one test case (also no need to reset all mocks usually, just clear them).